specs/validator/tableland-openapi-spec.yaml

```diff
@@ -201,7 +225,7 @@ components:
     TxnReceipt:
       type: object
       properties:
-        tableId:
+        table_id:
```

Comment:
As per our other discussion, we should switch this back to camelCase (sorry, I know this is still in draft mode).